Source/Extensions.swift

```diff
@@ -71,11 +71,7 @@ func delay(_ delay:Double, closure:@escaping ()->()) {
 
 internal extension Bundle {
     class func frameworkBundle() -> Bundle {
```

Comment:
Not strictly necessary, but you could make this a computed var instead of a func (because you don't have to pass any arguments).

Reply:
Oh! That makes sense 👍  fix on the way